src/gui/owncloudgui.cpp

```diff
@@ -97,12 +94,17 @@ ownCloudGui::ownCloudGui(Application *parent)
         this, &ownCloudGui::slotShowGuiMessage);
 }
 
+ownCloudGui::~ownCloudGui()
+{
+    _settingsDialog->deleteLater();
+}
```

Comment:
Why do we need this? Isn't the SettingsDialog parented to the ownCloudGui anyway?

Reply:
ownCloudGui is a QObject and the dialog a Widget so it doesn't work here.

Reply:
Huh? Can you elaborate a bit more?